flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStStateRequestClassifier.java

```diff
@@ -99,6 +99,20 @@ private void convertStateRequestsToForStDBRequests(StateRequest<?, ?, ?, ?> stat
                     dbPutRequests.add(forStMapState.buildDBBunchPutRequest(stateRequest));
                     return;
                 }
+            case AGGREGATING_GET:
+                {
+                    ForStAggregatingState<?, ?, ?, ?, ?> state =
+                            (ForStAggregatingState<?, ?, ?, ?, ?>) stateRequest.getState();
+                    dbGetRequests.add(state.buildDBGetRequest(stateRequest));
+                    return;
+                }
+            case AGGREGATING_PUT:
```

Comment:
Can be merged with `VALUE_UPDATE/LIST_UPDATE...`?

Reply:
done